docker/rviz/zed.rviz

```diff
@@ -0,0 +1,338 @@
+Panels:
```

Comment:
(minor) probably better for this to live with the other rviz files unless it makes it the docker setup harder